File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.cs
using System;

public class Solution 
{
    public int[] solution(int[] numbers, int num1, int num2) 
    {
        int[] resultList = new int[num2-num1+1];
        
        int index = 0;
        for(int i=num1; i<=num2; i++)
        {
            resultList[index] = numbers[i];
            index++;
        }
        return resultList;
    }
}

File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> numbers, int num1, int num2) 
{
    vector<int> answer(num2-num1+1);
    copy(numbers.begin()+num1, numbers.begin()+num2+1, answer.begin());

    return answer;
}